run_tools/run_edge_cube_extractor.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+"""Run Script for the EDGE CUBE extractor."""
+# ## IMPORTS ## #
+# Standard imports
+import os
+import sys
+import shutil
+
+# Third party imports
+from pathlib import Path
+from tqdm import tqdm
+
+
+# Local imports
+sys.path.append(".")
+sys.path.append("..")
+# pylint: disable=import-error,wrong-import-position
+from normits_demand import logging as nd_log
+from normits_demand.tools import edge_cube_extractor
+from normits_demand.utils import file_ops
+
+# pylint: enable=import-error,wrong-import-position
+
+# TODO (MI) Setup user inputs using BaseConfig
+
+# ## USER INPUTS ## #
+# cube catalogue setup
+CUBE_EXE = Path("C:/Program Files/Citilabs/CubeVoyager/VOYAGER.EXE")
+CUBE_CAT_PATH = Path(r"C:\Work\NorMITs\NorTMS_T3_Model_v8.16b")
+CAT_RUN_DIR = "Runs"
+CUBE_RUN_ID = "ILP_2018"
+
+# Input files
+TLC_OVERWRITE_PATH = Path(r"C:\Work\NorMITs\inputs\TLC_Overwrite_EDGE.csv")
+
+# Output location
+OUT_PATH = Path(r"C:\Work\NorMITs\outputs\Post_Review")
+
+# ## CONSTANTS ## #
+# logger
+LOG_FILE = "Export_BaseMatrices_Logfile.Log"
+LOG = nd_log.get_logger(f"{nd_log.get_package_logger_name()}.run_edge_cube_extractor")
+
+# Derived from inputs
+CUBE_CAT_RUN_PATH = Path(CUBE_CAT_PATH, CAT_RUN_DIR, CUBE_RUN_ID)
+
+# ## CLASSES ## #
+
+
+# ## FUNCTIONS ## #
+def run_extractor():
+    """Process Fixed objects."""
+    # time periods
+    periods = ["AM", "IP", "PM", "OP"]
+
+    # copy Cube files
+    for period in periods:
+        # read distance matrix
+        file_ops.check_file_exists(
+            Path(CUBE_CAT_RUN_PATH, f"Outputs/BaseAssign/{period}_stn2stn_costs.csv")

```

Comment:
You can use the `'/'` operator instead e.g. `CUBE_CAT_RUN_PATH / f"Outputs/BaseAssign/{period}_stn2stn_costs.csv"`